client/scripts/build_hospital_client.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```

Comment:
I would call this script `generate_client_env.sh` (because it does not build the client). Also, maybe we'll have to handle https someday so we'll have to make this script compatible (later).